src/index.Rmd

```diff
@@ -23,23 +23,44 @@ survival analysis to illustrate the main features of the package.
 Enjoy reading!
 
 ## Quick start
-A simple stratified cross-validation of [linear discriminant analysis](&MASS::lda) with [%mlr].
-```{r}
+Applying a statistical algorithm with [%mlr] has a basic structure of 3 steps not depending on the specific model. 
+
+```{r,results='hide',warning=FALSE,message=FALSE}
+
 library(mlr)
 data(iris)
 
-## Define the task
-task = makeClassifTask(id = "tutorial", data = iris, target = "Species")
+## 1) Define the task
+## specify the type of the analysis (e.g. classification), data and response 
+task = makeClassifTask(data = iris, target = "Species")
 
-## Define the learner
+## 2) Define the learner
+## choose a specific algorithm (e.g.linear discriminant analysis)
 lrn = makeLearner("classif.lda")
 
-## Define the resampling strategy
-rdesc = makeResampleDesc(method = "CV", stratify = TRUE)
+n = nrow(iris)
 
-## Do the resampling
-r = resample(learner = lrn, task = task, resampling = rdesc, show.info = FALSE)
+## 3) Fit the model
+## by combining learner and task 
```

Comment:
"Combining"  learner and task is a little unspecific.
Maybe sth. like: "Train the learner on the task"